component-interaction/src/app/userAge.pipe.ts

```diff
@@ -0,0 +1,10 @@
+import { Pipe, PipeTransform } from "@angular/core";
+
+@Pipe({
+	name: "getAge"
+})
+export class UserAgePipe implements PipeTransform {
+	public transform(dateOfBirth: any, ...args: any[]): number {
+		return 2022 - dateOfBirth?.getFullYear();

```

Comment:
```suggestion
		return 2022 - dateOfBirth?.getFullYear();

```
will this snippet work next year? :))

Reply:
😭😭😭🤦‍♂️